### PR TITLE
More various fixes

### PR DESCRIPTION
**Pull request by 0xC0ncord**
Fixes for various services including:

- Allowing several services to read certbot TLS certificates and keys
- Allow dovecot to tell spamd to learn spam or ham
- Let sysadm interact with usbguard at runtime
- Let sysadm interact with systemd-networkd
- Let sshd create polyinstantiated tmp directories for remote user logins
- and a couple others

### CODE REVIEW

File: policy/modules/services/bind.te [hunk 1]
```diff
@@ -81,7 +81,8 @@ allow named_t self:fifo_file rw_fifo_file_perms;
 allow named_t self:unix_stream_socket { accept listen };
 allow named_t self:tcp_socket { accept listen };
 
-allow named_t dnssec_t:file read_file_perms;
+filetrans_pattern(named_t, named_conf_t, dnssec_t, dir, "cache")
+allow named_t dnssec_t:file manage_file_perms;
```
Comment (pebenito) [inline policy/modules/services/bind.te:85]: Nit: the `allow` should be above the `filetrans_pattern`.

File: policy/modules/services/spamassassin.if [hunk 2]
```diff
@@ -196,6 +196,7 @@ interface(`spamassassin_exec_client',`
 
 	corecmd_search_bin($1)
 	can_exec($1, spamc_exec_t)
+	read_lnk_files_pattern($1, spamc_exec_t, spamc_exec_t)
```
Comment (pebenito) [inline policy/modules/services/spamassassin.if:199]: There should be no `spamc_exec_t` symlinks. The labeling should be fixed instead.

**Review by pebenito — CHANGES_REQUESTED** [review]

File: policy/modules/services/spamassassin.if (hunk 2 above)
Comment (0xC0ncord) [reply policy/modules/services/spamassassin.if:199]: OK. I will split that out into its own commit, since it is primarily for `rspamd`.

**Review by pebenito — APPROVED** [review]

**Comment by 0xC0ncord** [comment]
Fixed the documentation in the `certbot_read_lib` interface just now. Caught it at the last second.

**Pull request merged**